app/workers/fast_track/hypertension_special_issue_manager.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionSpecialIssueManager
+    attr_accessor :submission
+
+    def initialize(submission)
+      @submission = submission
+    end
+
+    def add_special_issue
+      data = JSON.parse(submission.form_json)
+      disabilities = data['form526']['form526']['disabilities']
+      added = add_rrd_to_disabilities(disabilities)
+      data['form526']['form526']['disabilities'] = added
```

Comment:
Should we combine these lines? I'm not sure the word `added` helps much here as the method already describes that.

Reply:
Fixed in https://github.com/department-of-veterans-affairs/vets-api/commit/6b517735d4d0c2f82a214bf09c177b004329077d.